contracts/ERC1155.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "./utils/Ownable.sol";
+
+
+// ERC1155Standard
+contract ERC1155Standard is ERC1155, Ownable {
```

Comment:
ERC1155Standard _is not ownable_ so we shouldn't export an ownable as a standard.

rename this contract to `ERC1155Ownable`